cogs/utils/settings.py

```diff
@@ -171,6 +175,26 @@ def set_server_mod(self, server, value):
         self.bot_settings[server.id]["MOD_ROLE"] = value
         self.save_settings()
 
+    def get_server_prefixes(self, server):
+        if server is None or server.id not in self.bot_settings:
+            return self.prefixes
+        p = self.bot_settings[server.id].get("PREFIXES")
```

Comment:
but why None? imo it should be the basecase (e.g. `[]`) or raise an exception. That way the function never gives a data type that people aren't expecting.